alerter.c

```diff
@@ -24,9 +36,22 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
+    // Testcase 1: Test networkAleter success
+    // simulate TestNetworAlertStatus for success
+    TestNetworAlertStatus = 200;
     alertInCelcius(400.5);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold.

Reply:
Agreed